lib/auth/oidc.go

```diff
@@ -769,8 +769,19 @@ func mergeClaims(a jose.Claims, b jose.Claims) (jose.Claims, error) {
 // getClaims gets claims from ID token and UserInfo and returns UserInfo claims merged into ID token claims.
 func (a *Server) getClaims(oidcClient *oidc.Client, connector types.OIDCConnector, code string) (jose.Claims, error) {
```

Comment:
We'll need test coverage for this function, take a look at how @timothyb89 added test coverage: https://github.com/gravitational/teleport/pull/7297

Reply:
added test